apollo-federation/src/connectors/json_selection/methods/public/as.rs

```diff
@@ -323,12 +323,12 @@ mod tests {
 
         assert_eq!(
             selection!("person->as", spec).shape().pretty_print(),
-            "Error<\"Method ->as requires one or two arguments (got 0)\", $root.person>",
+            "Error<\n  \"Method ->as requires one or two arguments (got 0)\",\n  $root.person,\n>",
```

Comment:
Maybe use `r#""#` syntax for those blocks with `\n`:

```rust
r#"Error<
  "Method ->as requires one or two arguments (got 0)",
  $root.person,
>"#
```

and get for free removing `\"`